template/html/edit_feed.html

```diff
@@ -92,6 +92,7 @@ <h3>{{ t "page.edit_feed.last_parsing_error" }}</h3>
     <div class="panel">
         <ul>
             <li><strong>{{ t "page.edit_feed.last_check" }} </strong><time datetime="{{ isodate .feed.CheckedAt }}" title="{{ isodate .feed.CheckedAt }}">{{ elapsed $.user.Timezone .feed.CheckedAt }}</time></li>
+            <li><strong>{{ t "page.edit_feed.next_check" }} </strong><time datetime="{{ isodate .feed.NextCheckAt }}" title="{{ isodate .feed.NextCheckAt }}">{{ .feed.NextCheckAt }}</time><br />({{ t "page.edit_feed.next_check_note" }})</li>
```

Comment:
I think it would be better to have this line shown only when the scheduler is configured to `entry_frequency`. Then the text "page.edit_feed.next_check_note" won't be necessary.

Reply:
I am ok to remove the note. (Instead, adding the notes to FAQ or the help)
But I think the next_check_at will still be helpful even when the scheduler is round robin. 
If a user sets the polling interval for a single feed (#949), but uses a incompatible global option, they may not get what they want.

Reply:
If we keep the next_check_at for all schedulers, the note is relevant. People may ask: why my feed is not fetched at the next_check_at. The note answers that question.
Let me know if you want to keep the notes or not.
Thanks,
Best,